test/github-issues/11285/issue-11285.ts

```diff
@@ -167,15 +167,19 @@ describe("github issues > #11285 Missing MSSQL input type", () => {
                         "query",
                     )
 
+                    const excludedUserIds = [user2.memberId]
```

Comment:
can you add a comment to this test about why you're creating and checking the array? If we refactor these tests at some point I think that context would be lost.